uberspace/roles/uberspace_ephios/templates/ephios.env.j2

```diff
@@ -3,19 +3,21 @@
 export SECRET_KEY={{ django_secret }}
 export DEBUG=False
 
-export DATABASE_URL=mysql://{{ ansible_user }}:{{ db_password }}@localhost/{{ ansible_user }}_ephios
+export DATABASE_URL={{ database_url }}
  
-export ALLOWED_HOSTS={{ domain }}
+export ALLOWED_HOSTS={{ domains|join(",") if use_main_documentroot else domain }}
 export STATIC_URL=/static/
-export STATIC_ROOT=/var/www/virtual/{{ ansible_user }}/{{ domain }}/static/
-export MEDIA_ROOT=/var/www/virtual/{{ ansible_user }}/{{ domain }}/media/
- 
+export STATIC_ROOT=/var/www/virtual/{{ ansible_user }}/{{ "html" if use_main_documentroot else domain }}/static/
+export STATIC_ROOT=/var/www/virtual/{{ ansible_user }}/{{ "html" if use_main_documentroot else domain }}/static/
+
+{% if not use_main_documentroot %}
 export SITE_URL=https://{{ domain }}
+{% endif %}
 
-export EMAIL_URL=smtp+tls://{{ smtp_mailbox }}@{{ domain }}:{{ smtp_password }}@{{ ansible_host }}:587
+export EMAIL_URL=smtp+tls://{{ smtp_mailbox }}@{{ smtp_domain if use_main_documentroot else domain }}:{{ smtp_password }}@{{ ansible_host }}:587
```

Comment:
this is currently breaking when we want to send mails for the ephios.de domain as the ansible_host is not allowed to do so. @felixrindt can you think of a nice way to pass in the password for the SMTP mailbox on our website uberspace?